content/sensu-core/0.29/platforms/sensu-on-microsoft-windows.md

```diff
@@ -34,10 +34,8 @@ installer package (i.e. a .msi file), which is available for download from the
 
 ### Download and install Sensu using the Sensu MSI {#download-and-install-sensu-core}
 
-_NOTE: As of Sensu version 1.0 repository URLs have changed.  To
-install or upgrade to the latest version of Sensu, please ensure you
-have updated existing configurations to follow the repository URL
-format specified below._
+_NOTE: As of Sensu version 1.0 repository URLs have changed.
```

Comment:
Let's say "as of version 0.29" in this document, use version 1.0 in all later versions